File: Задание 3.py
ttm=int(input())
mw= 10080
md = 1440
mh =60
weeks=ttm // mw
a = ttm % mw
days = a // md
a = a % md
hours = a // mh
minutes = a % mh
print(weeks)
print(days)
print(hours)
print(minutes)

File: Code/First_work/Задание 3.pas
const 
  mw= 10080;
  md = 1440;
  mh =60;
var ttm, minutes, hours, days, weeks, a:integer;

begin
  readln(ttm);
  weeks:=ttm div mw;
a := ttm mod mw;
days := a div md;
a := a mod md;
hours := a div mh;
minutes := a mod mh;
  writeln(ttm, ' ', weeks, ' ', days, ' ', hours, ' ', minutes, ' ');
end.
